app/posts/ecf-v2/initial-release-of-ecf-2.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Deciding what to build for the initial release of ECF 2
+description: "What we want to build to replace and improve the digital services that facilitate the early career framework (ECF) policy reforms"
+date: 2024-07-11
+---
+
+## What we've been working on
+
+Over the last two months, we have been working on a discovery to explore the biggest problems in the existing digital services that facilitate the early career framework (ECF) policy reforms.
+
+As part of this work, we had to define the minimum viable product for what we need to build to replace them. The minimum viable product, or MVP, is what we need to build for the new services, before we can initially release them. In this context, it's the minimum we need to make the services usable and fulfill their core purposes.
+
+We need a MVP because it's impossible to address all the problems or build all the solutions at once. It's better to release something early that users can test, which we can then improve. The smaller this initial release, or MVP, is, the sooner we can do this.
+
+Other new features and iterations will come to these services in the future.
+
+The services we considered in our discovery included:
```

Comment:
While all important, maybe we lead with Manage in this list as it will have greatest impact:

* Manage training for early career teachers
* The lead provider API and its documentation
* Internal finance and support tooling for DfE colleagues
* Check data for appropriate bodies
* Check data for delivery partners